x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/styles.ts

```diff
@@ -47,14 +47,15 @@ export const WaterfallChartFixedTopContainerSidebarCover = euiStyled(EuiPanel)`
 
 export const WaterfallChartFixedAxisContainer = euiStyled.div`
   height: ${FIXED_AXIS_HEIGHT}px;
+  z-index: ${(props) => props.theme.eui.euiZLevel5};
```

Comment:
Does this actually have to be level 5? I'm working on a feature now where unnecessarily high z-indexes caused a few issues, for example with overlapping flyouts. I'd prefer to use a lower z-index if a lower z-index would be appropriate here.

Reply:
i have reduced it to 4